python/app.py

```diff
@@ -0,0 +1,63 @@
+from flask import Flask, render_template, request
+import os
+from azure.ai.textanalytics import TextAnalyticsClient
+from azure.core.credentials import AzureKeyCredential
+from werkzeug.exceptions import HTTPException
+
+
+
+app = Flask(__name__)
+
+def create_text_analytics_client():
+    # Set a default key if missing to avoid TypeError on use
+    key = os.getenv('CS_ACCOUNT_KEY', '')
+    endpoint = f"https://{os.getenv('CS_ACCOUNT_NAME')}.cognitiveservices.azure.com/"
+    return TextAnalyticsClient(endpoint=endpoint, credential=AzureKeyCredential(key))
+
+def language_detection(text):
+    if not text:
+        return {}
+
+    client = create_text_analytics_client()
+    response = client.detect_language(documents=[text])
+    if response and response[0].primary_language:
+        return response[0]
+
+    return {}
+
+@app.route('/')
+def index():
+ 
+    text = request.args.get('text', '')
+    if not text:
+        return render_template('index.html', text="", language=None, error=None)
+
+    result = language_detection(text)
+    if result:
+        language = result.primary_language.name
+        confidence_score = result.primary_language.confidence_score
+        score_text = "--"
+        if confidence_score > 0.9:
+            score_text = "very confident"
+        elif 0.3 < confidence_score < 0.9:
```

Comment:
what about == 0.9? I think it might be less error prone to just do cascading > thans. Or at least make sure its inclusive of all values.

Reply:
fixed